src/Service.Host/client/src/components/supplierUtility/tabs/WhereTab.js

```diff
@@ -16,6 +16,7 @@ import countriesActions from '../../../actions/countriesActions';
 function WhereTab({
     orderAddressId,
     orderFullAddress,
+    invoiceAddressId,
```

Comment:
Was this prop already being passed in from the parent component and just unused?

Reply:
yeah it seemed like it was intended to be there - i needed it for the supplier display